.github/dependabot.yml

```diff
@@ -130,6 +135,11 @@ updates:
     schedule:
       interval: "daily"
 
+- package-ecosystem: "docker"
```

Comment:
The indentation is off here which is what's making pre-commit fail and say this isn't valid YAML.

Reply:
Fixed!